packages/perseus/src/widgets/graded-group/graded-group.tsx

```diff
@@ -105,7 +102,7 @@ type State = {
 // via defaultProps.
 0 as any as WidgetProps<
     PerseusGradedGroupWidgetOptions,
-    PerseusGradedGroupRubric
+    null
```

Comment:
Since the default for this type parameter is `Empty`, using `null` here is slightly inconsistent with the other widget types. I don't think it really matters, though. Maybe the default type arg should be `null` or `undefined`?

Reply:
I was thinking the same thing, I just always hate defining something as undefined. I'll make the change.